translations/de/pack/rar.de.json

```diff
@@ -0,0 +1,374 @@
+        "title": "Wilde Verfolgungsjagd"
+    },
+    {
+        "code": "22010",
+        "keywords": "Konsole",
+        "text": "+1[mu]\nWenn du zum ersten Mal in einem Zug einen erfolgreichen Run durchführst, darfst du 1[credit] erhalten und dir die oberste Karte deines Stapels ansehen. Falls du dies tust, darfst du jene Karte unter deinen Stapel legen.\nNur 1 <strong>Konsole</strong> pro Spieler.",
+        "title": "Paragon"
+    },
+    {
+        "code": "22011",
+        "flavor": "\"Das Aufkommen der Cryptowährung hat das Bankwesen deutlich verkompliziert. Die Finanzkriminalität hat mit der Entwicklung nicht nur Schritt halten können, im Gegenteil: sie ist geradezu explodiert.\"\n-Nach dem Flash: Die Geschichte eines Krieges, der nie stattfand.",
+        "text": "Immer wenn du einen erfolgreichen Run durchfährst, darfst du 1[credit] aus der Bank auf Finanzielle Mittel legen.\n[trash]: Nimm alle Credits von Finanzielle Mittel.",
+        "title": "Finanzielle Mittel"
+    },
+    {
+        "code": "22012",
+        "flavor": "\"Eigentlich ist jedes Ice nur eine Art Bezahlschranke.\"\n-Liza Talking Thunder",
+        "keywords": "Ice-Brecher - Fracter",
+        "text": "Immer wenn eine Begegnung mit einem Ice endet, in dem du Magnat verwendet hast, um eine Subroutine auszuschalten, erhält der Kon 2[credit].\n1[credit]: Schalte bis zu 3 Subroutinen einer <strong>Barriere</strong> aus.\n2[credit]: +3 Stärke.",
+        "title": "Magnat"
+    },
+    {
+        "code": "22013",
+        "flavor": "Ein wirklich stilvolles Alibi.",
+        "keywords": "Ort - Protzig",
+        "text": "Wenn du zum ersten Mal in einem Zug eine Markierung vermeidest oder entfernst, darfst du eine Karte aus deinem Griff installieren und die Installationskosten dafür um 1 senken.",
+        "title": "Thunder-Kunstgalerie"
+    },
+    {
+        "code": "22014",
+        "flavor": "\"Sollte ich es nicht ausschalten können, kenne ich jemanden, der es ganz bestimmt schafft.\"",
+        "keywords": "Kontakt",
+        "text": "Lege 12[credit] aus der Bank auf Miss Bones, sobald diese Karte installiert wird. Sobald keine Credits mehr auf Miss Bones liegen, zerstöre sie.\nVerwende diese Credits, um installierte Karten zu zerstören.",
+        "title": "Miss Bones"
+    },
+    {
+        "code": "22015",
+        "keywords": "Klon",
+        "text": "Immer wenn du auf Karten von F&E zugreifst, geben du und der Kon geheim 0[credit], 1[credit] oder 2[credit] aus. Deckt die ausgegebenen Credits auf. Falls du und der Kon dieselbe Anzahl Cedits ausgeben haben, greife auf 1 zusätzliche Karte von F&E zu.",
+        "title": "Akiko Nisei: Durchgedreht"
+    },
+    {
+        "code": "22016",
+        "keywords": "Verdoppeln",
+        "text": "Als zusätzliche Kosten, um dieses Ereignis zu spielen, gib [click] aus.\nDer Kon darf sich die obersten 4 Karten von F&E ansehen und sie in einer beliebigen Reigenfolge zurücklegen.\nDecke die obersten 4 Karten von F&E auf.",
```

Comment:
Alle anderen Doubles haben "Als zusätzliche Ausspielkosten für dieses Ereignis zahle [click]". Hm.

Reply:
Steht jedoch auf dieser Karte in diesem Wortlaut.

Reply:
Aua.